packages-backend/express/src/types.ts

```diff
@@ -25,6 +31,17 @@ export type TSessionMiddlewareOptions = {
 
   /* Options for the `jwksRsa.expressJwtSecret` */
   jwks?: Omit<ExpressJwtOptions, 'jwksUri'>;
+
+  // In order to determine the 'audience' for JWT validation, the middleware needs
+  // to resolve the request url.
+  // By default, the middleware assumes it receives a Node.js IncomingMessage object
+  // as its first parameter.
+  // This default behaviour might not work for every scenario as, for example,
+  // AWS lambda functions, where that first parameter is an Event object.
+  // (Reference: https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-develop-integrations-lambda.html)
+  // To handle these situations, this option allows the delegation of the URL resolution
+  // from the request-like object provided to the middleware as its first parameter.
```

Comment:
We already explain the AWS example in the docs, I think we can keep this comment simple as to what the function is for. 

```suggestion
  // By default we assume that the `request` is a Node.js-like object having either
  // a `originalUrl` or `url` properties.
  // If that's not the case (for example in AWS Lambda functions) you need to correctly
  // map the URL (URI path + query string) of the `request`.
```

Reply:
Updated in this commit e4be0ae17a4678626bf426a1c7ab237e458a00d1